src/main.rs

```diff
@@ -119,7 +113,7 @@ fn game_loop() {
     info!("done! cpu: {}", screeps::game::cpu::get_used())
 }
 
-fn cleanup_memory() -> Result<(), Box<::std::error::Error>> {
+fn cleanup_memory() -> Result<(), Box<dyn ::std::error::Error>> {
```

Comment:
I think you don't need the `::` in front of `std` anymore with rust 2018.

Reply:
right!